docs/Installation.md

```diff
@@ -402,20 +402,27 @@ su -l
 Install dependencies available from binary packages:
 
 ```sh
-pkg install p5-Class-Method-Modifiers p5-Config-IniFiles p5-Daemon-Control p5-DBI p5-File-ShareDir p5-File-Slurp p5-HTML-Parser p5-IO-CaptureOutput p5-JSON-PP p5-JSON-RPC p5-Moose p5-Parallel-ForkManager p5-Plack p5-Plack-Middleware-Debug p5-Role-Tiny p5-Router-Simple p5-Starman p5-String-ShellQuote
+pkg install p5-Class-Method-Modifiers p5-Config-IniFiles p5-Daemon-Control p5-DBI p5-File-ShareDir p5-File-Slurp p5-HTML-Parser p5-IO-CaptureOutput p5-JSON-PP p5-JSON-RPC p5-Moose p5-Parallel-ForkManager p5-Plack p5-Plack-Middleware-Debug p5-Role-Tiny p5-Router-Simple p5-Starman p5-String-ShellQuote net-mgmt/p5-Net-IP-XS databases/p5-DBD-SQLite devel/p5-Log-Dispatch devel/p5-Log-Any devel/p5-Log-Any-Adapter-Dispatch
 ```
 
-Optionally install Curl (only needed for the post-installation smoke test)
+Optionally install Curl (only needed for the post-installation smoke test):
 
 ```sh
 pkg install curl
 ```
 
+Verify remaining dependencies:
+
+> This step is not needed for installation. It is only here for the developers of Zonemaster.
+
+```sh
+cpanm --scandeps Zonemaster-Backend-x.y.z.tar.gz # Normally you skip this step.
+```

```

Comment:
This step should include an instruction on what to do with the command output. I.e. that the modules listed in the following `cpanm` command should match the set of direct dependencies in the output, and that transitive dependencies should be ignored.

Also, the step belongs in the ReleaseProcess document, rather than here in Installation.

Reply:
I agree that this is not the best place. As the release process document is set up there is no obvious place for it, but I'll remove it from here and create an issue to integrate it into some updated document.